docs/development/build-steps.md

```diff
@@ -178,6 +178,17 @@ step.
 Read more at [`Configuration#Shared source files`](configuration.md#shared-source-files).
 ```
 
+### Run Command
+
+* Identifier: **`runCommand`**
+* Implementation: [`Builder\Generator\Step\RunCommandStep`](https://github.com/CPS-IT/project-builder/blob/main/src/Builder/Generator/Step/RunCommandStep.php)
+
+The `runCommand` enables you to execute custom shell commands in the generated project directory.
+Since this steps requires a `BuildResult` it makes most sense to execute your command after dependent steps have
```

Comment:
Every step requires a `BuildResult` which makes this sentence probably a bit misleading. Can you rephrase it?